app/src/main/res/values/strings.xml

```diff
@@ -18,10 +18,11 @@
     <string name="name">Name</string>
     <string name="other">Other</string>
     <string name="edit_profile">Edit Profile</string>
-    <string name="logout">Logout</string>
+    <string name="logout">Logout</string> 
+    <string name="add_post">Share Your Story</string> 
```

Comment:
take a look

Reply:
i use line 22 in my code, "share your story"